### DISCUSSION TITLE
React Query included in the ReactJS 2020 Community Survey

### DISCUSSION THREAD

**Discussion by banagale** [TanStack/query#1142] [General]
<img width="647" alt="Screen Shot 2020-10-06 at 4 10 27 PM" src="https://user-images.githubusercontent.com/1409710/95269546-774bcf80-07ee-11eb-8fd4-e3b4896f8539.png">

[Reactjs.org](https://reactjs.org/) has a 2020 Community Survey up. One question includes includes React Query:

### What do you currently use for data-fetching with React (check all that apply)?

I thought people here might want to be sure to fill it out so they get represented in the survey results, so head to the site, the link for the survey is in a header banner above.

I couldn't help but notice that the previous question:  **What do you currently use to manage state with React (check all that apply)?** does not include RQ.

I checked off the normal react state but I also threw React Query in other, since I think I'm substituting use of state with RQ in some cases. That said, I'm pretty new to React.

Here's [a screenshot of the two questions](https://user-images.githubusercontent.com/1409710/95270207-e8d84d80-07ef-11eb-87cb-921ed23d29b4.png) in the survey I mention above.

**Comment by TkDodo** [comment]
> but I also threw React Query in other

Ha, I did the same :) I read the first question and was a bit disappointed that react-query was missing 😅
